torch/storage.py

```diff
@@ -826,6 +827,13 @@ def hpu(self, device=None, non_blocking=False, **kwargs) -> T:  # type: ignore[m
         hpu_storage: torch.UntypedStorage = self._untyped_storage.hpu(device, non_blocking, **kwargs)
         return self._new_wrapped_storage(hpu_storage)
 
+    def xpu(self, device=None, non_blocking=False, **kwargs) -> T:  # type: ignore[misc, type-var]
+        _warn_typed_storage_removal()
+        if self.dtype in [torch.quint8, torch.quint4x2, torch.quint2x4, torch.qint32, torch.qint8]:
+            raise RuntimeError("Cannot create XPU storage with quantized dtype")
+        xpu_storage: torch.UntypedStorage = self._untyped_storage.xpu(device, non_blocking, **kwargs)
```

Comment:
for MYPY issue `"UntypedStorage" has no attribute "xpu" `, please refer to ["UntypedStorage" has no attribute "xpu" ](https://github.com/pytorch/pytorch/blob/88fbe79550b41e94d4b78bb5be3afff10cb0ef99/torch/storage.py#L41)

Reply:
I fixed it locally. Will push the update as soon as we will close on bc-linter story.

Reply:
pushed a fix.